incubator/cms-xcache/cms-xcache/templates/deployment.yaml

```diff
@@ -100,9 +100,9 @@ spec:
             value: "{{ .Values.XCacheConfig.Prefetch }}"
           - name: XC_ROOTDIR
             value: "{{ .Values.XCacheConfig.LocalRoot }}"
-          - name: WQ_BLOCKS_PER_LOOP
+          - name: XC_WQ_BLOCKS_PER_LOOP
```

Comment:
This file is for a different application `cms-xcache`. Did you mean to edit it?

Reply:
reverted... just to be on a safe side. Please keep in mind that the new OSG image will have these variables with "XC_" prepended.